CHANGELOG.rst

```diff
@@ -2,6 +2,175 @@
 Release Notes
 =============
 
+.. _Release Notes_6.13.0:
+
+6.13.0
+======
+
+.. _Release Notes_6.13.0_Prelude:
+
+Prelude
+-------
+
+Release on: 2019-07-24
+
+- Please refer to the `6.13.0 tag on integrations-core <https://github.com/DataDog/integrations-core/blob/master/AGENT_CHANGELOG.md#datadog-agent-version-6130>`_ for the list of changes on the Core Checks
+
+
+.. _Release Notes_6.13.0_Upgrade Notes:
+
+Upgrade Notes
+-------------
+
+- APM: Services are no longer aggreagated in the agent, nor written to the Datadog API.
+  Instead, they are now automatically extracted on the backend based on the received
+  traces.
+
+- The ``port`` option in the NTP check configuration is now parsed as an integer instead of a string.
+
+
+.. _Release Notes_6.13.0_New Features:
+
+New Features
+------------
+
+- APM: add support for Unix Domain Sockets by means of the `apm_config.receiver_socket` configuration. It is off by default. When set,
+  it must point to a valid sock file.
+
+- APM: API emitted metrics now have a lang_vendor tag when the Datadog-Meta-Lang-Vendor
+  HTTP header is sent by clients.
+
+- APM: Resource-based rate limiting in the API can now be completely
+  disabled by setting `apm_config.max_memory` and/or `apm_config.max_cpu_percent`
+  to the value 0.
+
+- Add support for environment variables in checks' config files
+  using the format "%%env_XXXX%%".
+
+- Add new systemd integration to monitor systemd itself
+  and the units managed by systemd.
+
+- The total number of bytes received by dogstatsd is now reported by the
+  `dogstatsd-udp/Bytes` and `dogstatsd-uds/Bytes` expvar.
+
+- Adds the ability to use `DD_TAGS` to set global tags in Fargate.
+
+- Added a support for the new pod log directory pattern introduced in version 1.14 of Kubernetes to make sure
+  the agent keeps on collecting logs after upgrade of a Kubernetes cluster.
+
+
+.. _Release Notes_6.13.0_Enhancement Notes:
+
+Enhancement Notes
+-----------------
+
+- Add a kube_cronjob tag in the tagger. It applies to container metrics, autodiscovery metrics and logs.
+
+- Change the prefix of entity IDs to make it easier to query the tagger
+  without knowing what the container runtime is.
+
+- APM: reduce memory usage in high traffic by up to 10x.
+
+- APM: The default interval at which the agent watches its resource usage has
+  been reduced from 20s to 10s.
+
+- APM: Improved processing concurrency and as a result, CPU usage decreased
+  by 20% in some scenarios.
+
+- APM: Queued sender was rewritten to improve performance around scenarios where network problems are present.
+
+- APM: Code clean up around configuration and writer.
+
+- The `datadog-agent` command now prints the version of Golang the agent was
```

Comment:
I believe it's the `datadog-agent version` command (see #3717). People might be confused as to what's the difference between this change and the next one otherwise (`Display Go version in output of status command`).